kolibri/utils/options.py

```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+
+from configobj import ConfigObj
+from configobj import flatten_errors
+from configobj import get_extra_values
+from validate import Validator
+
+
+logger = logging.getLogger(__name__)
+
+ini_specification = [
+
+    "[Database]",
+    "DATABASE_ENGINE = option('sqlite', 'postgres', default='sqlite')",
+    "DATABASE_NAME = string(default='')",
+    "DATABASE_PASSWORD = string(default='')",
+    "DATABASE_USER = string(default='')",
+    "DATABASE_HOST = string(default='localhost')",
+
+    "[Paths]",  # these are defined relative to KOLIBRI_HOME (or absolute)
+    "CONTENT_ROOT = string(default=content)",
+
+    "[Deployment]",
+    "LISTEN_PORT = integer(default=8080)",
```

Comment:
maybe `HTTP_PORT`? (wondering if there may eventually be other ports like websockets or https)

Reply:
This is for direct backwards compatibility to support `KOLIBRI_LISTEN_PORT`. We *could* add a new primary name for this, but then we would need to code in legacy support and figure out precedence rules if both env vars were defined, etc. Worth it? :man_shrugging:

Reply:
Update to PR will have a clean way to handle legacy envvars.